docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -419,14 +419,109 @@ As discussed in treatment guideline reviews, in general triplet regimens are pre
+
+This model strategy does NOT consider:
+
+- Differences between isatuxamib-containing treatment regimens (ex: isatuxamib + pomalidomide + dexamethasone versus isatuxamib + carfilzomib + dexamethasone)
+- Differences between the numerous treatment regimens that do NOT contain isatuxamib (note this has important implications for applying efficacy data to the appropriate reference group(s))
+- Specific eligibility requirements for isatuxamib-contianing treatments other than previous treatment with isatuxamib. Specifically, 
+	- We do not directly consider the requirement of previous treatment with lenalidomide and a PI, although this is likely minimally impactful as these are very common treatment regimens and "almost all" patients with myeloma develop disease that is eventually refracotory to lenalidomide and PIs, [Attal-et-al-2019]_.
+	- We do not consider previous treatment with other monoclonal antibody drugs such as daratumumab, which may impact probability of receiving an isa-containing treatment
+
+We will consider the following treatment categories in our treatment model:
+
+- Isa-containing treatment regimen
+- Non-isa-containing treatment regimen
+
+.. note::
+
+	Other treatment categories to consider tracking include: reference treatment regimen from ISA RCT (polidamide + dexamethasone), daratumumab-containing regimens
+
+Importantly, we will assume that a given percentage of patients who receive isa-containing regimen for a specific line of treatment will not receive an isa-containing regimen at a later line of treatment.
+
+	Currently, we will assume that 100% of patients who receive isa-containing regimens will not receive isa-containing regimens at a later line of treatment.
+
+.. todo::
+	
+	Confirm use of probability 1.0 for ineligibility for future isa-containing treatment regimen based on past treatment with isa.
+
+	Alternative approach: make ineligible if simulant progresses on or within 60 days seven months time OR with a probability of 60% if simulant progresses within seven months. This was chosen because the ISA trial defined non-response as progression on or within 60 days or progression within 6 months after achieving at least a partial response and reported at least partial response in 60% of subjects and a median time to response among this group was 35 days).
+
+.. note::
+
+	Consider similarly assuming that patients who recieve dara-containing regimens will not receive isa-containing regimens at later lines of treatment.
+
+The overall proportion of patients who receive isa-containing treatment regimens at each line of treatment and in each scenario is shown in the table below.
```

Comment:
I assume overall proportion stands for percent of prevalent cases plus percent of incident cases, correct?

Reply:
Hmm, I think we want this to be incident cases. I'll update this.